rel-eng/packages/atomic-reactor

```diff
@@ -1 +1 @@
-1.6.23.1-1 ./
+1.6.23-1 ./
```

Comment:
latest release version is 1.6.23.2

so not sure if there should be future version or current version